### ISSUE TITLE
loss: nan

### ISSUE THREAD

**Issue by antoniosap** [fferroni/DEC-Keras#9]
HI,

i have this value.
Any ideas?

**Comment by dipanjannag** [comment]
Facing same issue with simple code
```
c = DeepEmbeddingClustering(n_clusters=10, input_dim=17)
c.initialize(X, finetune_iters=100, layerwise_pretrain_iters=500)
c.cluster(X, y=labels)
```
the loss is quickly reaching `nan` even before one full iteration

**Comment by ganaz** [comment]
check if any column has all zero value and remove them , the another thing you can do is reduce learning rate. I think  by reducing learning rate may fix this issue.
